packages/wonder-blocks-modal/components/standard-modal.test.js

```diff
@@ -6,18 +6,10 @@ import StandardModal from "./standard-modal.js";
 
 describe("StandardModal", () => {
     test("Clicking the close button triggers `onClickCloseButton`", () => {
```

Comment:
The test title no longer reflects what the test is doing.

Reply:
Ah, will fix.